finagle-core/src/main/scala/com/twitter/finagle/tracing/Id.scala

```diff
@@ -26,7 +26,7 @@ object SpanId {
       val s = "%02x".format(bb)
       Array(s(0), s(1))
     }
-  ).toArray
+   ).toArray
```

Comment:
nit: is this a stray whitespace addition?

Reply:
Yeah man, this IntelliJ setting (as well as format on save) keeps getting reset when I reopen IntelliJ and the change has snuck past me a few times now.

I'll edit the whitespace changes in vim. Thanks for point it out. Sorry about that.